<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ascii Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actNewBitmap"/>
    <addaction name="actOpenBitmap"/>
    <addaction name="actSaveBitmap"/>
    <addaction name="actSaveBitmapAs"/>
    <addaction name="actExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actResizeBitmap"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actOpenBitmap">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actSaveBitmap">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actNewBitmap">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actResizeBitmap">
   <property name="text">
    <string>Resize Bitmap</string>
   </property>
  </action>
  <action name="actExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actSaveBitmapAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
